torch/_dynamo/variables/torch.py

```diff
@@ -1189,3 +1189,23 @@ def torch_function_override_enabled(self, tx, args, kwargs):
                 (torch._ops.OpOverload, torch._ops.OpOverloadPacket),
             )
         ) and can_dispatch_torch_function(tx, args, kwargs)
+
+
+class TorchDispatchKeySetVariable(BaseTorchVariable):
+    """represents torch.DispatchKeySet"""
+
+    @classmethod
+    def create_with_source(cls, value, source):
+        install_guard(source.make_guard(GuardBuilder.ID_MATCH))
+        return cls(value, source=source)
+
+    def call_method(
+        self,
+        tx,
+        name,
+        args: "List[VariableTracker]",
+        kwargs: "Dict[str, VariableTracker]",
+    ) -> "VariableTracker":
+        if name == "highestPriorityTypeId":
+            return variables.EnumVariable(self.value.highestPriorityTypeId())
```

Comment:
Is there some way to treat these things as constant and have dynamo guard on them? That looks like what is being done here, but I want to know if there's a more reusable way we can do this.

Reply:
Do you mean an API like `mark_constant`, which would wrap the return value as ```ConstantVariable``` or ```EnumVariable``` and guard on them automatically? I think we can add a helper function.